privacy-security-explainer.md

```diff
@@ -57,9 +60,182 @@ Rounding, quantization, and fuzzing are three categories of mitigations that mod
+
+Once a session is created, developers may attempt to create a bounded reference space by passing `unbounded` into `XRSession.requestReferenceSpace()`.
+
+```js
+let xrReferenceSpace;
+function onSessionCreated(session) {
+  session.requestReferenceSpace('unbounded')
+  .then((referenceSpace) => { xrReferenceSpace = referenceSpace; })
+  .catch( (e) => { /* handle gracefully */ } );
+}
+```
+
+### Bounded reference spaces
+Bounded reference spaces, by themselves, do not enable developers to determine geographic location. However, because the floor level is established and users are able to walk around, it may be possible for a site to infer the user’s height or perform gait analysis, allowing user profiling and fingerprinting. In addition, it may be possible perform fingerprinting using the bounds reported by a bounded reference space.
+
+Developers indicate the desire for bounded viewer tracking at the time of session creation by adding `bounded-floor` to either `XRSessionInit.requiredFeatures` or `XRSessionInit.optionalFeatures`. 
+
+```js
+function onVRClick() {
+  xr.requestSession('immersive-vr', { requiredFeatures: ['bounded-floor'] } )
+  .then(onVRSessionCreated);
+}
+
+xr.requestSession('inline', { optionalFeatures: ['bounded-floor'] } )
+  .then(onInlineSessionCreated);
+}
+```
+
+In response, the UA must ensure that:
+* The document is allowed to use all the policy-controlled features associated with the sensor types used to track the native origin of a bounded reference space
+* User intention is well understood, either via [explicit consent](#explicit-consent) or [implicit consent](#implict-consent)
+* The device is capable of bounded tracking
+
+If these requirements are not met and `bounded-floor` is listed in `XRSessionInit.requiredFeatures` then the promise returned from `requestSession()` must be rejected. Otherwise, the promise may be fulfilled but future calls to `XRSession.requestReferenceSpace()` must fail when passed `bounded-floor`.
+
+Once a session is created, developers may attempt to create a bounded reference space by passing `bounded-floor` into `XRSession.requestReferenceSpace()`.
+
+```js
+let xrReferenceSpace;
+function onSessionCreated(session) {
+  session.requestReferenceSpace('bounded-floor')
+  .then((referenceSpace) => { xrReferenceSpace = referenceSpace; })
+  .catch( (e) => { /* handle gracefully */ } );
+}
+```
+
+In response, the UA must ensure that: 
+* Bounded reference spaces are allowed to be created based on the restrictions above
+* Any group of `local`, `local-floor`, and `bounded-floor` reference spaces that are capable of being related to one another must share a common native origin; this restriction does not apply when `unbounded` reference spaces are also able to be created
```

Comment:
I'm actually not 100% certain this addresses the motion controller issue (and realized as much over the weekend) as there is no firm technical requirement that a motion controller share a native origin with anything.  Either way it might be useful to have a dedicated section for other `XRSpace` types and start it off with `XRInputSource`.  I'm gonna poke at this a bit this morning and see if I can come up with something good.

Reply:
Tracked in #752